service/gcs/core/gcs/devices.go

```diff
@@ -0,0 +1,105 @@
+package gcs
+
+// TODO: JTERRY75 this is from runc/devices.go. When we fix the vendor issue pull this in from that pacakge instead.
+
+import (
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/sys/unix"
+)
+
+var (
+	ErrNotADevice = errors.New("not a device node")
+)
+
+// Testing dependencies
+var (
+	unixLstat     = unix.Lstat
+	ioutilReadDir = ioutil.ReadDir
+)
+
+// Given the path to a device and its cgroup_permissions(which cannot be easily queried) look up the information about a linux device and return that information as a Device struct.
```

Comment:
nit: line-wrap comment

Reply:
This isnt our code. Leaving it alone since I will be removing.

Reply:
I missed this was copied code on the first pass.